arrow-fx/src/main/kotlin/arrow/fx/Queue.kt

```diff
@@ -67,13 +67,13 @@ interface Dequeue<F, A> {
   fun peekAll(): Kind<F, List<A>>
 }
 
-/** A polymoprhic effect typeclass that allows [Dequeue]'ing values from a [Queue]. */
+/** A polymorphic effect typeclass that allows [Enqueue]'ing values from a [Queue]. */
```

Comment:
```suggestion
/** A polymorphic effect typeclass that allows [Enqueue]'ing values to a [Queue]. */
```
Missed that on the first read through